bin/utils/processFolder.js

```diff
@@ -19,9 +19,20 @@ const processFolder = (input, output, stylesheet) => {
         processFile(file, output, stylesheet);
       });
 
+    const mdFiles = files.filter((file) => path.extname(file) === ".md");
```

Comment:
We could combine with the 2 filters for .md and .txt into 1 filter only
eg: `const supportedFiles = files.filter((...)=> ... === ".txt" || ... === ".md")`